jsonnet/components/alertmanager.libsonnet

```diff
@@ -88,6 +88,9 @@ function(params)
           },
         ],
         type: 'ClusterIP',
+        selector+: {
+          app:: null,
```

Comment:
shouldn't this be done in kube-prometheus directly?

Reply:
agree, we had better change it in kube-prometheus.
i'm going to make a pull request to kube-prometheus on this modification. meanwhile, we keep it here until upstream changes.